nifi-nar-bundles/nifi-aws-bundle/nifi-aws-processors/src/main/java/org/apache/nifi/processors/aws/kinesis/stream/ConsumeKinesisStream.java

```diff
@@ -292,27 +316,23 @@ public class ConsumeKinesisStream extends AbstractKinesisStreamProcessor {
             )
     );
 
-    private static final Map<String, PropertyDescriptor> DISALLOWED_DYNAMIC_KCL_PROPERTIES = new HashMap<String, PropertyDescriptor>(){{
-        put("regionName", REGION);
-        put("timestampAtInitialPositionInStream", STREAM_POSITION_TIMESTAMP);
-        put("initialPositionInStream", INITIAL_STREAM_POSITION);
-        put("dynamoDBEndpoint", DYNAMODB_ENDPOINT_OVERRIDE);
-        put("kinesisEndpoint", ENDPOINT_OVERRIDE);
-        put("failoverTimeMillis", FAILOVER_TIMEOUT);
-        put("gracefulShutdownMillis", GRACEFUL_SHUTDOWN_TIMEOUT);
+    private static final Map<String, PropertyDescriptor> DISALLOWED_DYNAMIC_KCL_PROPERTIES = new HashMap<>() {{
+        put("leaseManagementConfig.initialPositionInStream", INITIAL_STREAM_POSITION);
+        put("leaseManagementConfig.failoverTimeMillis", FAILOVER_TIMEOUT);
     }};
 
     private static final Object WORKER_LOCK = new Object();
-    private static final String WORKER_THREAD_NAME_TEMPLATE = ConsumeKinesisStream.class.getSimpleName() + "-" + Worker.class.getSimpleName() + "-";
+    private static final String SCHEDULER_THREAD_NAME_TEMPLATE = ConsumeKinesisStream.class.getSimpleName() + "-" + Scheduler.class.getSimpleName() + "-";
 
     private static final Set<Relationship> RELATIONSHIPS = Collections.singleton(REL_SUCCESS);
     private static final Set<Relationship> RECORD_RELATIONSHIPS = Collections.unmodifiableSet(new HashSet<>(Arrays.asList(REL_SUCCESS, REL_PARSE_FAILURE)));
 
     private static final PropertyUtilsBean PROPERTY_UTILS_BEAN;
     private static final BeanUtilsBean BEAN_UTILS_BEAN;
+
     static {
         PROPERTY_UTILS_BEAN = new PropertyUtilsBean();
-        PROPERTY_UTILS_BEAN.addBeanIntrospector(new FluentPropertyBeanIntrospector("with"));
+        PROPERTY_UTILS_BEAN.addBeanIntrospector(new FluentPropertyBeanIntrospector(""));
```

Comment:
If we no longer expet/have a `with` prefix, do we need the `BeanUtils` at all, or will straight up reflection work for the direct field names?

Reply:
BeanUtils still gives us the benefit of setting the value properly based on the type.

Reply:
That's fair, but is the `FluentPropertyBeanIntrospector` still required, or can we just use the `PROPERTY_UTILS_BEAN` instance without a `BeanIntrospector` being added (or is there a better alternative)?